server/src/test/java/org/kaazing/gateway/server/config/parse/ServiceDefaultsTest.java

```diff
@@ -26,7 +30,7 @@ public void testDefaultConnectOptions() throws Exception {
                 .connectOption("udp.interface", "en0")
                 .connectOption("tcp.transport", "socks://localhost:8000")
                 .connectOption("http.keepalive", "disabled")
-                .connectOption("http.keepaliveTimeout", "5sec")
+                .connectOption("http.keepalive.timeout", "5sec")
```

Comment:
The name wasn't changed, this is the configuration value that goes in the XML.  Based on how I changed things the ConfigBuilder uses those values that were in the XML rather than the camel case names.

I removed the strongly typed accept/connect options from the XSD, but didn't change any other behavior.  I was assuming an end user wouldn't notice and thus didn't think there was doc impact.  Correct me if I'm wrong.